utils/db_utils.py

```diff
@@ -486,8 +498,150 @@ def query_segments_crossing_endpoints(poly_region_start, poly_region_end, start_
 # The following query can be called multiple times, let's open db only once
 analysis_timeseries_db = edb.get_analysis_timeseries_db()
 
+
 # Fetches sensed_mode for each section in a list
 # sections format example: [{'section': ObjectId('648d02b227fd2bb6635414a0'), 'user_id': UUID('6d7edf29-8b3f-451b-8d66-984cb8dd8906')}]
 def query_inferred_sections_modes(sections):
     return esds.cleaned2inferred_section_list(sections)
 
+
+def query_entries_chunked(key_list, date_query,
```

Comment:
The current version of `query_entries_chunked` does not meet the requirements:

> This function will basically be a wrapper around `ts.find_entries` with the chunking logic around it. It should essentially take the same arguments as `find_entries`:
> - key_list
>   - for trajectories, that key was analysis/recreation_location or background/location. In your generic function, it could be anything, and that gets passed to `ts.find_entries`
> - time_query
> - geo_query
> - extra_query_list
> - the query limit (default = 250k)
> 
> return a list (or iterator) of all the combined entries (which may be > 250k)

Namely, it has `date_query` instead of `time_query`, and it yields and returns dataframes instead of simply returning a list.